Source/Core/writeTextToCanvas.js

```diff
@@ -1,9 +1,95 @@
-import measureText from "../ThirdParty/measureText.js";
 import Color from "./Color.js";
 import defaultValue from "./defaultValue.js";
 import defined from "./defined.js";
 import DeveloperError from "./DeveloperError.js";
 
+function measureText(context2D, textString, font, stroke, fill) {
+  var metrics = context2D.measureText(textString);
```

Comment:
Let's avoid settings our own properties on the `TextMetrics` object like the previous implementation did. Instead create a separate object with the properties `width`, `height`, `ascent`, `descent`, `minx` and return that.